src/ngx-gallery-preview.component.ts

```diff
@@ -48,6 +49,7 @@ export class NgxGalleryPreviewComponent implements OnChanges {
     @Input() showDescription: boolean;
     @Input() swipe: boolean;
     @Input() fullscreen: boolean;
+    @Input() previewFullSize: boolean;
```

Comment:
If you initialize this component via `[fullSize]="currentOptions?.previewFullSize"` you can use `fullSize` as variable.